### PR TITLE
is function checked and allowed after if got error

### PR DESCRIPTION
**Pull request by testbetter200**

### CODE REVIEW

File: src/apiCall.js [hunk 1]
```diff
@@ -175,6 +175,18 @@ module.exports = function apiCall(file, apiPort, itApi = it) {
...
         } catch (err) {
+          if (req.after && _.isFunction(req.after)) {
+            req.after.call(req, {
+              apiPort,
+              specs: config.apiCalls.swagger,
+              res: {},
+              expect,
+              op: operations[req.call],
+              body: allReqData,
+              error: err,
```
Comment (mateusfreira) [inline src/apiCall.js:187]: Can we update the readme with this parameter?

File: src/customTypes/faker.js [hunk 2]
```diff
@@ -74,6 +74,10 @@ function isFaker(data) {
 }
 
 function evaluateFaker(data, scope) {
+  if (isFunction(data)) {
+    return data
```
Comment (mateusfreira) [inline src/customTypes/faker.js:78]: ❤️

**Review by mateusfreira — CHANGES_REQUESTED** [review]

File: src/apiCall.js (hunk 1 above)
Comment (dreich-nucleus) [reply src/apiCall.js:187]: Also, would it be possible to write a unit test that verifies that the after call is made when there is an exception?

**Review by dreich-nucleus — COMMENTED** [review]
Looks fine - I would love to see a new unit test but only if that is not too difficult. Also, adding to the README is a great idea.

File: src/apiCall.js (hunk 1 above)
Comment (dreich-nucleus) [reply src/apiCall.js:187]: Also, in the readme, be sure to document that the `res` field will be an empty object. Perhaps it should be `null` or even undefined?

Comment (testbetter200) [reply src/apiCall.js:187]: updated the readme only for now

File: src/apiPort.js [hunk 3]
```diff
@@ -75,6 +77,20 @@ class ApiPort {
       })
     }
 
+    if (process.env.COMMON_DATA_CONFIG) {
```
Comment (mateusfreira) [inline src/apiPort.js:80]: This change has broken the CI build(check https://travis-ci.org/testbetter/openapitest/builds/580571263?utm_source=github_status&utm_medium=notification), Also this should be committed on another branch, once it is not related to the open PR is function checked and allowed after if got error #15

Comment (testbetter200) [reply src/apiPort.js:80]: sry I missed that. Fixed :)

**Review by mateusfreira — APPROVED** [review]
LGTM, let's open an issue do cover the apiPort.js with test

**Pull request merged**